service/telemetry/config.go

```diff
@@ -91,6 +94,14 @@ type LogsConfig struct {
 	InitialFields map[string]interface{} `mapstructure:"initial_fields"`
 }
 
+// LogsSamplingConfig sets a sampling strategy for the logger. Sampling caps the
+// global CPU and I/O load that logging puts on your process while attempting
+// to preserve a representative subset of your logs.
+type LogsSamplingConfig struct {
```

Comment:
Is it worth adding a note in the troubleshooting docs as to how to enable sampling? https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#logs

Reply:
We have only minimal examples there, not sure if we should add all of them there, or keep it minimal only the level.

Reply:
We can do it in a separate PR.